games-strategy/hedgewars/hedgewars-0.9.24.1.ebuild

```diff
@@ -41,7 +41,21 @@ CDEPEND="
 	!libav? ( media-video/ffmpeg:= )"
 DEPEND="${CDEPEND}
 	>=dev-lang/fpc-2.4
-	dev-qt/linguist-tools:5"
+	dev-qt/linguist-tools:5
+	server? (
+		>=dev-lang/ghc-6.10
+		dev-haskell/vector
+		>=dev-haskell/network-2.3
+		dev-haskell/random
+		>=dev-haskell/mtl-2
+		dev-haskell/sandi
+		dev-haskell/hslogger
+		dev-haskell/utf8-string
+		dev-haskell/sha
+		dev-haskell/entropy
+		>=dev-haskell/zlib-0.5.3 <dev-haskell/zlib-0.6
+		dev-haskell/regex-tdfa
+	)"
```

Comment:
Would you mind sorting the deps in the `server?` dependency block alphabetically? This might sound like nitpicking but IMHO it improves maintainability of the ebuild quite a bit.

Reply:
it's now alphabetically sorted